handlers/handler-publickey-auth/src/main/java/org/apache/servicecomb/authentication/provider/RSAProviderTokenManager.java

```diff
@@ -50,7 +52,11 @@ public boolean valid(String token) {
         LOGGER.error("token is expired");
         return false;
       }
-      if (validatedToken.contains(rsaToken)) {
+      validatedToken = CacheBuilder.newBuilder()
```

Comment:
This logic seems totally wrong, basic tests should be taken.

Reply:
under modify

Reply:
ok